src/Microsoft.AspNet.OData.Shared/Delta.cs

```diff
@@ -43,6 +43,17 @@ public abstract class Delta : DynamicObject, IDelta
         /// <returns>True if the Property was found</returns>
         public abstract bool TryGetPropertyValue(string name, out object value);
 
+        /// <summary>
+        /// Attempts to add the delta nested resource.
+        /// </summary>
+        /// <param name="name">Name of the nested resource.</param>
+        /// <param name="deltaNestedResource">The delta object for the nested resource.</param>
+        /// <returns></returns>
+        public virtual bool TryAddNestedResource(string name, object deltaNestedResource)
```

Comment:
>TryAddNestedResource [](start = 28, length = 20)

I don't think this is necessary.

Reply:
Yup. Removed.

---
In reply to: [190748341](https://github.com/OData/WebApi/pull/1445#discussion_r190748341) [](ancestors = 190748341)

Reply:
It should be "undo", not comment them out.

Besides, if there's nothing change, please exclude it from the commit.

---
In reply to: [190772293](https://github.com/OData/webapi/pull/1445#discussion_r190772293) [](ancestors = 190772293,190748341)